scrapy_zyte_api/handler.py

```diff
@@ -34,6 +34,9 @@ def _load_retry_policy(settings):
     return policy
 
 
+_CLIENT_CACHE: Dict[Tuple[Optional[str], str, int], AsyncClient] = {}
```

Comment:
I think it'd be worth noting the issue #58 here as a comment, in case it gets refactored/removed later.